<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>moveShapeClass</class>
 <widget class="QWidget" name="moveShapeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>moveShape</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>851</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="chosenButton">
      <item>
       <property name="text">
        <string>ball</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>triangle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>rectangle</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pauseButton">
      <property name="text">
       <string>pause</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stopButton">
      <property name="text">
       <string>stop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="decelerateButton">
      <property name="text">
       <string>decelerate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="accelerateButton">
      <property name="text">
       <string>accelerate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="moveshape.qrc"/>
 </resources>
 <connections/>
</ui>
